media/js/firefox/ios.js

```diff
@@ -0,0 +1,145 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+;(function($, Mozilla) {
+    'use strict';
+
+    var $body = $('body');
+    var $html = $('html');
+    var COUNTRY_CODE = '';
+
+    // Countries in which Firefox for iOS is available.
+    // Add two-letter country codes (lowercase) to this array to
+    // to expand the market. We'll remove this logic for wide release.
+    // Note: This is the country code, NOT locale code; they often differ.
+    var market_countries = ['nz'];
+
+    // Get country code via geo-ip lookup
+    function getGeoLocation () {
+        try {
+            COUNTRY_CODE = geoip_country_code().toLowerCase();
+        } catch (e) {
+            COUNTRY_CODE = '';
+        }
+    }
+
+    $script('//geo.mozilla.org/country.js', function() {
```

Comment:
Probably not. This was copied from the [Firefox OS page](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/os/firefox-os.js#L164)